server/authorize.go

```diff
@@ -33,13 +32,13 @@ func (s *IDPServer) serveAuthorize(w http.ResponseWriter, r *http.Request) {
 
 	redirectURI := uq.Get("redirect_uri")
 	if redirectURI == "" {
-		http.Error(w, "tsidp: must specify redirect_uri", http.StatusBadRequest)
+		writeHTTPError(w, r, http.StatusBadRequest, "invalid_request", "tsidp: must specify redirect_uri")
```

Comment:
why are some error messages prefixed with "tsidp: " and others not?
this prefix might not mean much for the average user